src/js/components/TextInput/__tests__/__snapshots__/TextInput-test.js.snap

```diff
@@ -2640,6 +2717,84 @@ exports[`TextInput should not have accessibility violations 1`] = `
 </div>
 `;
 
+exports[`TextInput should not have padding when plain="full" 1`] = `
+.c0 {
+  font-size: 18px;
+  line-height: 24px;
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+}
+
+.c2 {
+  box-sizing: border-box;
+  font-size: inherit;
+  font-family: inherit;
+  border: none;
+  -webkit-appearance: none;
+  background: transparent;
+  color: inherit;
+  width: 100%;
+  padding: 11px;
```

Comment:
How come the snapshot shows the TextInput still have padding when plain is full.

Reply:
The padding is originally set to 11px but then modified to be 0. Look at line 2745 in the snapshot: `padding: 0;`